tweets_analyzer.py

```diff
@@ -0,0 +1,105 @@
+import argparse
+import os
+
+from source.get_user_tweets import download_user_tweets
+from source.user_sentiment_analysis import ascii_plots, png_plots
+
+
+class ArgumentError(Exception):
+    """
+    Username in twitter is required for sentiment analysis
+    """
+
+
+def make_arguments_parser():
+    parser = argparse.ArgumentParser(
+        description="Application for sentiment analysis of tweets published since the beginning of 2021"
+    )
+
+    parser.add_argument(
+        "-u",
+        "--username",
+        dest="username",
+        help="Twitter username for sentiment analysis",
+        required=False,
+        type=str,
+    )
+    parser.add_argument(
+        "-p",
+        "--plot",
+        dest="plot",
+        help="Type of bar charts with user statistics (png/ascii). Default - ascii.",
+        required=False,
+        default="png",
+        type=str,
+    )
+    parser.add_argument(
+        "-m",
+        "--model",
+        dest="model",
+        help="Choose desired model: glm (for logistic regeression model) or nn (for LSTM neural network model). Default - glm.",
+        required=False,
+        default="glm",
+        type=str,
+    )
+    parser.add_argument(
```

Comment:
Probably you could use https://docs.python.org/3/library/argparse.html#mutual-exclusion here.